modules/parent-join/src/internalClusterTest/java/org/elasticsearch/join/query/ChildQuerySearchIT.java

```diff
@@ -1306,40 +1307,23 @@ public void testParentChildQueriesNoParentType() throws Exception {
         client().prepareIndex("test").setId(parentId).setSource("p_field", "1").get();
         refresh();
 
-        try {
```

Comment:
I don't think we need to fix these? For the exception case the production code should handle any ref counting?
Maybe a cleanup best left for another PR since it's unrelated to this effort?